<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApplication</class>
 <widget class="QMainWindow" name="MainApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UtpPoker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="distributeButton">
          <property name="text">
           <string>Rozdaj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeCardsButton">
          <property name="text">
           <string>Wymieniam karty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmExchangeButtn">
          <property name="text">
           <string>Wymie≈Ñ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="notChangeCardsButton">
          <property name="text">
           <string>Nie wymieniam kart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="text">
           <string>Info:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
